src/components/major/Splash.js

```diff
@@ -42,8 +42,8 @@ const styles = StyleSheet.create({
     left: 0,
     bottom: 0,
     right: 0,
-    resizeMode: Constants.expoConfig.splash.resizeMode,
-    backgroundColor: Constants.expoConfig.splash.backgroundColor,
+    resizeMode: Constants.expoConfig.splash?.resizeMode,
+    backgroundColor: Constants.expoConfig.splash?.backgroundColor,
```

Comment:
I don't know expo, but are these values missing from your Expo [app.json/app.config.js](https://docs.expo.dev/versions/latest/config/app/#splash)? These constants have been used since Splash.js was created, so they appear to expect them to be set